blog/konferencja-tlumaczy-2024/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Konferencja tłumaczy 2024 - patronujemy!'
+authors: niedzwiedzka
+date: '2024-06-25'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'konferencje'
+  - 'tlumaczenia'
+---
+Kolejny rok i kolejna [Konferencja Tłumaczy (KTLC)](https://konferencjatlumaczy.pl/). Już od kilku lat jesteśmy jej dumnym patronem - nie inaczej jest w tym roku. I nie tylko patronem - uczestniczymy w konferencji (czasami nawet jako prelegenci i prelegentki) i możemy ją wam polecić z absolutnym spokojem o wysoką merytoryczną jakość oraz świetną atmosferę. 
+
+<!--truncate-->
+
+#### **Gdzie i kiedy?**
+
+Tegoroczna edycja odbędzie się 28 września w Warszawie, a towarzyszyć jej będzie Translation and Localization Conference – dwudniowe wydarzenie, zaplanowane na 27-28 września. Podobnie jak w ubiegłym roku, konferencja będzie miała formę hybrydową: stacjonarnie w Golden Tulip Warsaw Center oraz online na platformie Gridaly. Organizatorzy zadbali także o imprezy towarzyszące - warto przybyć do Warszawy już w czwartek, 26 września, wypić powitalnego drinka i wziąć udział w panelu dyskusyjnym. W niedzielę natomiast, po zakończeniu części merytorycznej, na uczestniczki i uczestników czeka gra miejska i zwiedzanie stolicy. 

```

Comment:
Może warto rozdzielić sekcję "Gdzie i kiedy?" na dwie sekcje:
- Gdzie i kiedy?
- Atrakcje dla uczestników (albo coś w tym stylu)

Reply:
Konferencja to tak naprawdę dwa wydarzenia, o czym piszesz wyżej. Są dwie osobne strony - konferencjatlumaczy.pl i translation-conference.com
Jeśli chodzi o listę prelegentów, to jest ona inna dla polskiej konfy (https://konferencja-tlumaczy.pl/?page_id=2121) i inna dla zagranicznej konfy (https://translation-conference.com/ktlc2024-speakers/). Myślę, że trzeba to jakoś mocniej podkreślić w tekście, żeby nie było wątpliwości czego gdzie szukać.